src/fontra/views/editor/panel-transformation.js

```diff
@@ -7,13 +7,17 @@ import {
 import { EditBehaviorFactory } from "./edit-behavior.js";
 import Panel from "./panel.js";
 import * as html from "/core/html-utils.js";
-import { rectFromPoints, rectSize, unionRect } from "/core/rectangle.js";
+import {
+  filterPathByPointIndices,
+  getSelectionByContour,
+} from "/core/path-functions.js";
+import { rectCenter, rectFromPoints, rectSize, unionRect } from "/core/rectangle.js";
 import {
   Transform,
```

Comment:
Unrelated, but `Transform` and `decomposedFromTransform` are not used, might as well clean up now.

Reply:
Transform is used. All other unused imports are removed, now.